### PR TITLE
ganyoa lesson08 exercise

### PR DESCRIPTION
**Pull request by ganyoa**
***please read before merge***   I'm not sure how/why but I have a previous students files showing as deletions in this pull request. His directory is showing up on my $ git status queries as 'untracked files'.  I'm not sure how to remove his directory from my git status, and I certainly do not want to impact any of his work.  Please let me know what steps to take to not impact his work and merge my lesson08.  I won't add this pull request to the assignment page for grading yet in case I have to change something.  I apologize for my mistake.  

Anthony Ganyo

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
before you start work you should always sync with master, this is usually the reason for what you're seeing.

**Comment by natasha-aleksandrova** [comment]
you should be able to follow this process here:
https://uwpce-pythoncert.github.io/PythonCertDevel/modules/GitWorkflow.html?highlight=workflow#the-usual-process

go ahead and merge with upstream and let's see if it resolves your issue.

**Comment by ganyoa** [comment]
[AG git issue.pdf](https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/files/4325689/AG.git.issue.pdf)

Please see attached screenshots from git bash. I think its showing my fork is synced with the master. However that students folder is still missing from my fork, but is available on my PC. And Im not sure why his folder is showing as 'untracked files'.

**Comment by natasha-aleksandrova** [comment]
@ganyoa did you sync with upstream per my comment above?

**Comment by ganyoa** [comment]
[ganyoa git issue2.pdf](https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/files/4330737/ganyoa.git.issue2.pdf)
Yes. At least I attempted to. I attached screenshots to my earlier post...is there something noticeable I'm doing wrong? I just tried again (additional screenshots attached) and now I have more student folders as "untracked", and an error during the pull upstream master.  I'm also still unsure what to do about Nick_Lenssen folder that is showing delete on my pull request. His folder is on my PC but not my fork.

**Comment by natasha-aleksandrova** [comment]
@ganyoa unfortunately this is difficult to debug remotely, you can give me write access to your fork and I can fix it for you (by adding me as a collaborator). But you would then need to reclone your fork again.

Another option is to just not merge your changes in given that you're almost done anyway.

File: students/ganyoa/lesson08/circle_class_exercise.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    def __str__(self): #step 6
+        return 'Sphere with radius: {}'.format(self.radius)
+
+    @property
+    def volume(self):
+        return ((4/3) * pi * self.radius**3)
+
+    @property
+    def area(self):
+        return (4 * pi * self.radius**2)
```
Comment (natasha-aleksandrova) [inline students/ganyoa/lesson08/circle_class_exercise.py]: looks great!

**Comment by natasha-aleksandrova** [comment]
@ganyoa don't forget to submit on the course website please

**Pull request closed without merging**